site/app/views/email/EmailStatusView.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace app\views\email;
+
+use app\views\AbstractView;
+use app\models\User;
+use app\libraries\FileUtils;
+use app\models\EmailStatusModel;
+
+class EmailStatusView extends AbstractView {
+    public function showEmailStatus($emailStatus) {
+        $this->core->getOutput()->addBreadcrumb("Email", $this->core->buildCourseUrl(["email"]));
+        $this->core->getOutput()->addVendorJs('bootstrap/js/bootstrap.bundle.min.js');
+        $this->core->getOutput()->addInternalCss('bootstrap.css');
```

Comment:
This `css` file is already included in all pages by Output class. This can be removed?